lib/libutils/ext/include/compiler.h

```diff
@@ -247,4 +247,6 @@
 #define __compiler_atomic_store(p, val) \
 	__atomic_store_n((p), (val), __ATOMIC_RELAXED)
 
+#define barrier() __asm__ __volatile__ ("" : : : "memory")
```

Comment:
Please write `asm volatile` instead. Not that it matters much, but that's how it's done at other places in OP-TEE.